.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: required
+
+services:
+  - docker
+
+script:
+  - git clone https://github.com/KDD-OpenSource/32de-python.git
```

Comment:
A bit inefficient, isn't it? I would suggest creating a new "deployment" repo.

Reply:
I would postpone the efficiency of travis builds for a moment.

Reply:
That's the start of things never getting done...